Aptoide/appcoins-wallet-android
# [MOB-955] Amplitude integration;

**What does this PR do?**

This PR implements amplitude analytics in the wallet

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AmplitudeAnalytics.kt

**How should this be manually tested?**

You should be able to see events going out through logs, just search for amplitude and every event we have for rakam should show up there as well

**What are the relevant tickets?**

  Tickets related to this pull-request: [MOB-955](https://aptoide.atlassian.net/browse/MOB-955)

**Code Review Checklist**
